account_invoice_rounding/account.py

```diff
@@ -158,125 +152,64 @@ def _compute_swedish_rounding(self, cr, uid, invoice, amounts,
                                                 context=ctx)
         return {}
 
-    def _amount_all(self, cr, uid, ids, name, args, context=None):
+    @api.one
+    @api.depends('invoice_line.price_subtotal', 'tax_line.amount')
```

Comment:
i've found the same dependencies on account module.
https://github.com/OCA/OCB/blob/8.0/addons/account/account_invoice.py#L64